form.js

```diff
@@ -1,26 +1,32 @@
-(function () {
-  "use strict";
-  const root = document.getElementById("root");
-  const form = document.createElement("form");
+const root = document.getElementById("root");
+const form = createNewEl('form');
 
+function createNewEl(el){
+  return document.createElement(el);
+}
```

Comment:
Let's use function declaration in the places where we need to work with context. Use arrow functions instead to be more consistent.